Library/Homebrew/keg.rb

```diff
@@ -190,6 +190,57 @@ def self.find_some_installed_dependents(kegs)
     [all_required_kegs.to_a, all_dependents.sort]
   end
 
+  # Given an array of kegs, this method finds casks dependent on them
+  # except those included in the `casks` array. If it does, it returns:
```

Comment:
I think it'd be good to have this somewhere under `cask/` rather than in `keg` (which is already largish)

Reply:
Actually, it might be possible to merge this method with `Keg.find_some_installed_dependents` as some of the code is repetitive. Would that be okay too?

Reply:
I'm 👍🏻 on sharing code on them if it makes sense. I don't think there should be any references to casks in `keg.rb` though; casks never have/use kegs.